synapse/storage/signatures.py

```diff
@@ -22,9 +22,7 @@
 from synapse.util.caches.descriptors import cached, cachedList
 
 
-class SignatureStore(SQLBaseStore):
-    """Persistence for event signatures and hashes"""
-
+class SignatureWorkerStore(SQLBaseStore):
     @cached()
     def get_event_reference_hash(self, event_id):
         return self._get_event_reference_hashes_txn(event_id)
```

Comment:
this appears to be broken. Perhaps we should remove it rather than moving it.

Reply:
Oh, good spot